admin_guide/pruning_resources.adoc

```diff
@@ -283,9 +306,39 @@ $ oadm prune images --keep-tag-revisions=3 --keep-younger-than=60m --confirm
 $ oadm prune images --prune-over-size-limit --confirm
 ----
 
+.Using secure or insecure connection
+
+The secure connection is preferred for production use. It is done over HTTPS
```

Comment:
Suggestion:
The secure connection is the preferred and recommended approach.

It's the recommended as a rule, not only for production.